bears/planemo/PlanemoLintBear.py

```diff
@@ -0,0 +1,29 @@
+
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.PipRequirement import PipRequirement
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+
+
+@linter(executable='planemo',
+        output_format='regex',
+        output_regex=r'..\s(?P<severity>\w*:)(?P<message>.*)',
+        severity_map={'WARNING:': RESULT_SEVERITY.MAJOR,
+                      'CHECK:': RESULT_SEVERITY.NORMAL,
+                      'INFO:': RESULT_SEVERITY.INFO})
+class PlanemoLintBear:
+    """
+    Checks the code with planemo lint. This will run
+    planemo lint over each file separately.
+    """
+    LANGUAGES = {'galaxy'}
+    REQUIREMENTS = {PipRequirement('planemo', '0.36')}
```

Comment:
You should have mentioned that instead ;) Also add it to `PINNED_PACKAGES` in `generate_bear_requirements.py` @Shade5